web-marketplace/src/lib/queries/react-query/wasm/wasmCodeDetails.helpers.ts

```diff
@@ -0,0 +1,220 @@
+import { fromBase64, toHex } from '@cosmjs/encoding';
+import {
+  QueryCodeRequest,
+  QueryCodeResponse,
+} from 'cosmjs-types/cosmwasm/wasm/v1/query';
+
+const UNKNOWN_QUERY_PATH_REGEX = /unknown query path/i;
+
+export type CodeDetailsLike = {
+  id: number;
+  checksum: string;
+  creator: string;
+  data?: Uint8Array;
+};
+
+export type WasmCodeClient = {
```

Comment:
why not just using `CosmWasmClient` from '@cosmjs/cosmwasm-stargate', we do not need to rewrite types which already exist there